src/layouts/AllServices.tsx

```diff
@@ -46,12 +46,12 @@ const AllServices = ({ Footer }: AllServicesProps) => {
 
   const fetchNavigation = async () => {
     const fetchNav = await fetchNavigationFiles();
-    const filteredBundles = await Promise.all(fetchNav.filter(({ id }) => availableBundles.includes(id)));
-    return filteredBundles;
+    const filteredBundles = await Promise.resolve(fetchNav.filter(({ id }) => availableBundles.includes(id)));
```

Comment:
I don't think this has to be a promise:
```js
const filteredBundles = fetchNav.filter(({ id }) => availableBundles.includes(id));
```
Based on the types the `fetchNav` is just an array of objects. Unless the types are wrong.

Reply:
oops yup you're right